### PR TITLE
Fix prop types warning messages

### PR DESCRIPTION
**Pull request by jenniemc**
## Summary
The google analytic event label `vaos-error`  are recorded on universal analytics (360) but there is not enough information from the GA dashboard to determine what caused the error or where it originates from. 

Despite not able to identify the GA bug, I did identify areas for improvement eliminating prop type warning and UI warning messages. 

### prop types and UI warning messages

When attempting to reproduce failure at cancel an appointment

Screenshot 2023-07-21 at 1 16 50 PM

Screenshot 2023-07-21 at 1 17 22 PM

Screenshot 2023-07-21 at 1 18 06 PM

When attempting to reproduce failure at fetching appointment slots

Screenshot 2023-07-21 at 1 22 46 PM

When attempting to reproduce failure at requesting a CC appointment

Screenshot 2023-07-21 at 1 24 49 PM

Screenshot 2023-07-21 at 1 26 15 PM

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#53873

## Testing done

n/a

## What areas of the site does it impact?
vaos

### CODE REVIEW

File: src/applications/vaos/appointment-list/components/cancel/CancelAppointmentFailedModal.jsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default function CancelAppointmentFailedModal({
           contact your medical center to cancel:
         </p>
       )}
-      <p>
+      <div>
```
Comment (vbahinwillit) [inline src/applications/vaos/appointment-list/components/cancel/CancelAppointmentFailedModal.jsx:37]: Why the change?

**Review by vbahinwillit — COMMENTED** [review]
LGTM!

**Review by vbahinwillit — APPROVED** [review]

File: src/applications/vaos/appointment-list/components/cancel/CancelAppointmentFailedModal.jsx (hunk 1 above)
Comment (jenniemc) [reply src/applications/vaos/appointment-list/components/cancel/CancelAppointmentFailedModal.jsx:37]: Changing the <p> tag to <div> fixes the warning message that <h5> cannot be a descent of a paragraph tag. The <h5> tag is generated from `FacilityAddress` function which is nested inside the now updated <div> tag .
